sigs/sig-marketing/social-media-guidelines.md

```diff
@@ -0,0 +1,51 @@
+**<span style="text-decoration:underline;">Assets</span>**
+
+[Argo Proj Social Content Calendar](https://docs.google.com/spreadsheets/d/1ysidts6KNAr2OKS1L4bgMToUFuAmO1xQ__tTQf4kPWk/edit?usp=sharing) (to prevent duplicate posting)
+
+**<span style="text-decoration:underline;">Guidelines*</span>**
+
+_Access_
+
+
+
+* Members of sig-marketing will have access to Argo Proj Social Accounts (LinkedIn, Twitter, and Youtube)
+* All social media posts by members related to the Argo Project whether from official, company or personal accounts must follow the below guidelines.
+* Posts and content should be approved by a member from a different organization before being published. 
+* For posts that do not follow guidelines: 2 members from **_different_** organizations must agree that a post should be taken down and then it will be deleted.
+
+_Objectives_
+
+
+
+* Raising awareness of events and talks about/for the community 
+    * Includes: ArgoCon, KubeCon, GitOpsCon, Argo Contributor Summit, CNCF Workshops, CNCF Certifications, Community Events
+* Empowering the DevOps community to use and leverage Argo Projects
+* Conversion: Communicating with the existing Argo community on new features and updates
+
+_LinkedIn/Twitter Posts_
+
```

Comment:
The goal of the project social accounts is to promote the project and related content. Any content posted, or linked to, should be relevant, no more than a week old and not be re-posts. 
Linking to vendor blogs and/or content should be restricted to once / month / vendor.